shanoir-ng-datasets/src/main/java/org/shanoir/ng/processing/carmin/service/CarminDatasetPorcessingServiceImpl.java

```diff
@@ -0,0 +1,89 @@
+package org.shanoir.ng.processing.carmin.service;
+
+import java.util.Optional;
+
+import org.shanoir.ng.processing.carmin.model.CarminDatasetProcessing;
+import org.shanoir.ng.processing.carmin.repository.CarminDatasetProcessingRepository;
+import org.shanoir.ng.shared.core.service.BasicEntityServiceImpl;
+import org.shanoir.ng.shared.event.ShanoirEvent;
+import org.shanoir.ng.shared.event.ShanoirEventService;
+import org.shanoir.ng.shared.event.ShanoirEventType;
+import org.shanoir.ng.shared.exception.EntityNotFoundException;
+import org.shanoir.ng.utils.KeycloakUtil;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+/**
+ * @author KhalilKes
+ */
+@Service
+public class CarminDatasetPorcessingServiceImpl extends BasicEntityServiceImpl<CarminDatasetProcessing>  implements CarminDatasetProcessingService {
+
+        @Autowired
+        private CarminDatasetProcessingRepository carminDatasetProcessingRepository;
+
+        @Autowired
+        private ShanoirEventService eventService;
+
+        @Override
+        protected CarminDatasetProcessing updateValues(CarminDatasetProcessing from, CarminDatasetProcessing to) {
```

Comment:
I would propose to make that method private please to better respect the principle of information hiding

Reply:
Hi Khalil, I had a second look and you can just leave it. Thank you!